C/common/reading.cpp

```diff
@@ -590,6 +604,20 @@ vector<Datapoint *> *values = new vector<Datapoint *>;
 			DatapointValue dpv(nestedValues, true);
 			values->push_back(new Datapoint(name, dpv));
 		}
+		else if (itr->value.IsArray())
+		{
+			vector<double> arr;
+			for (auto& v : itr->value.GetArray())
+			{
+				if (v.IsNumber())
+					arr.push_back(v.GetDouble());
+				else
+					throw runtime_error("Only numeric lists are currently supported in datapoints");
```

Comment:
are we catching this exception anywhere?

Reply:
Caller is supposed to handle this exception like other existing runtime_error